### PR TITLE
Add the Template file

### LINKED ISSUE #33: Template for formatting
I have just drafted a Lecture_template.md file that I will make a pull request for. I was thinking we could all agree, once pending PRs are approved, that we each had until a certain day to edit the template, then we should conform to the one format, more or less. 

I propose, once PRs are approved, we give it two days to edit the template as we see fit, then anything in main can be pulled by anyone for formatting.

### PR DESCRIPTION
**Pull request by superslalomracer**
This is for the purpose of internal use to help with formatting and editing of each other's code. 

Add markdown file to be used as a template for review and all
future notes. Open for edits until a date specified in the
related issue. After the file is final, all notes can be
edited to conform.

Template file describes convention for using example code as
both python and bash code, details an organization based on
main topics and a general commands at the end.

### CODE REVIEW

File: Lecture_template.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+Text-body 
+...  
+...  
+...  
+...  
+...  
+put two spaces for clean line breaks
+
+```
+python example code(foo)
```
Comment (kevinaangstadt) [inline Lecture_template.md:17]: GitHub md has a way of specifying the language for the block of code (e.g., ```python) 

Please update accordingly

File: Lecture_template.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+**Maybe this one is a list of steps or commands**
+
+* listed item
+* listed item
+* **command**: explanation
+* **command -flag**: explanation
+
+#### Topic 3
+
+```
```
Comment (kevinaangstadt) [inline Lecture_template.md]: Same as Line 22, you can specify the language of this code block

File: Lecture_template.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
+#### CS 340 Class Notes Summer 2021
+# Lecture #: Topic   
+(mm/dd/2021)
+
+## **Overview**
```
Comment (kevinaangstadt) [inline Lecture_template.md]: Issue #33 discussion suggests removing this section

**Review by kevinaangstadt — CHANGES_REQUESTED** [review]
Please address the comments in this review.

**Comment by superslalomracer** [comment]
All requested changes have been made. New PR will be opened shortly.

**Review by kevinaangstadt — APPROVED** [review]

**Pull request merged**